src/ChatFooter.js

```diff
@@ -16,16 +16,18 @@ export default class ChatFooter extends React.Component {
       const { scope } = this.props;
 
       if (scope === "round") {
-        const { stage, player } = this.props;
+        const { stage, player, chatId } = this.props;
         stage.append("chat", {
```

Comment:
This is not what this PR is about, but I do wonder, why is the "round" scope saving data on the `stage`.

Reply:
I didn't even notice, but that does sound like something to fix